components/bl-treemap-component/README.md

```diff
@@ -0,0 +1,23 @@
+# Treemap
+Treemap is a component of Backendless UI-Builder designer. The main purpose of the component is proportional display of data. Also, the [react-d3-treemap](https://github.com/jquintozamora/react-d3-treemap) library is used to create this component.
```

Comment:
Please, take a look at this library

![image](https://user-images.githubusercontent.com/15347646/193777438-15ff7d8b-37ba-482b-bcdd-1f104afeb8df.png)

I don't think it's enough level of code quality to share it with customers which pay for our services.

Please, found the more popular solution with a rich API, or write it by yourself - to have the possibility of delivering new features.